app/utils/mail/mailworker.py

```diff
@@ -38,14 +43,16 @@ def send_email(
     msg["To"] = ";".join(recipient)
     msg["Subject"] = subject
 
-    if file_path:
-        with Path.open(Path(file_path), "rb") as f:
-            msg.add_attachment(
-                f.read(),
-                main_type=main_type,
-                sub_type=sub_type,
-                filename=file_path.split("/")[-1],
-            )
+    # Not working
+    # if file_directory and file_name:
+    #     file_path = Path(file_directory, str(file_uuid))
+    #     with Path.open(file_path, "rb") as file:
+    #         msg.add_attachment(
+    #             file.read(),
+    #             main_type=main_type,
+    #             sub_type=sub_type,
+    #             filename=file_name,
+    #         )
```

Comment:
I think you can remove this code. You could maybe create an issue to keep track of this issue